tests/utilities/test_introspection_from_schema.py

```diff
@@ -56,3 +61,11 @@ def converts_a_simple_schema_without_description():
             }
             """
         )
+
+    def benchmark_executing_introspection_query(
```

Comment:
My suggestion here is to move this to a different file, just like in GraphQL.js.

We could name it `benchmark_introspection_from_schema.py` in the same directory, or move it to a subdirectory `tests/benchmarks/test_introspection_from_schema.py` which I like even better. then you can run the benchmarks with `pytest tests/benchmark --benchmark-enable`, pytest doesn't need to sift through the whole suite to filter the benchmarks, and we have all the benchmarks in one place clearly arranged.